core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -825,6 +826,22 @@ default HttpResponse mapError(Function<? super Throwable, ? extends Throwable> f
         return of(HttpMessage.super.mapError(function));
     }
 
+    /**
+     * Applies specified {@link Consumer} to the {@link ResponseHeaders} emitted by this {@link HttpResponse}.
+     *
```

Comment:
Let's also add
`This method is a shortcut for {@code peek(action, ResponseHeaders.class)}.`